docs/spark/how-to-guides/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Deploy Worker and UDF Binaries
+description: Learn how to deploy Worker and UDF (User-Defined Function) binaries.
+ms.date: 12/30/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Deploy Worker and UDF Binaries
+
+This how-to provides general instructions on how to deploy Worker and UDF (User-Defined Function) binaries, 
```

Comment:
Should worker be capitalized? Also, this sentence should only be on one line instead of 3.

```suggestion
This how-to provides general instructions on how to deploy .NET for Apache Spark worker and user-defined function binaries, including which Environment Variables to set up and some commonly used parameters when launching applications with `spark-submit`.
```